pos_default_empty_image/static/src/xml/pos_default_empty_image.xml

```diff
@@ -14,7 +22,7 @@
                     </span>
                 </t>
             </div>
-            <div class="product-name-without-image" t-attf-id="article_product_#{product.id}">
+            <div class="product-name-without-image" t-attf-id="article_product_#{product.id}" t-attf-style="font-size: #{font_size}px !important;">
```

Comment:
To avoid large spacings between lines, I would suggest experimenting with `line-height`, something like so:
```suggestion
            <div class="product-name-without-image" t-attf-id="article_product_#{product.id}" t-attf-style="font-size: #{font_size}px !important; line-height: #{font_size}px !important;">
```
I didn't check this, just wrote this suggestion.

Reply:
done. thanks.